react/components/SKUSelector/Wrapper.js

```diff
@@ -1,12 +1,37 @@
-import React, { useContext } from 'react'
+import React, { useContext, useMemo } from 'react'
 import { ProductContext } from 'vtex.product-context'
 import { pathOr } from 'ramda'
 
 import SKUSelector from './index'
 
+const useVariations = (skuItems, shouldNotShow) => {
+  const result = useMemo(() => {
+    if (shouldNotShow) {
+      return {}
+    }
+    const variations = {}
+    for (const skuItem of skuItems) {
+      for (const currentVariation of skuItem.variations) {
+        const { name, values } = currentVariation
+        const value = values[0]
+        const currentSet = variations[name] || (new Set())
+        currentSet.add(value)
+        variations[name] = currentSet
+      }
+    }
+    const variationsNames = Object.keys(variations)
+    // Transform set back to array
+    for (const varName of variationsNames) {
+      const set = variations[varName]
+      variations[varName] = Array.from(set)
+    }
+    return variations
+  })
```

Comment:
you are using `useMemo`, but are not setting its dependencies. this renders its usage moot afaik

Reply:
its a mistake, thanks